pkg/config/types/v1alpha1/v1alpha1_types.go

```diff
@@ -510,9 +510,17 @@ type EtcdConfig struct {
 type ClusterNetworkConfig struct {
 	//   description: |
 	//     The CNI used.
-	//   values:
-	//     - flannel
-	CNI string `yaml:"cni"`
+	//     Composed of "name" and "url".
+	//     URL is only used if name == "remote".
+	//     URL should point to a single yaml file that will get deployed.
```

Comment:
I think this needs to be updated since its a slice now, and not a single string.